config/custom-environment-variables.yml

```
@@ -6,6 +6,12 @@ public:
   metrics:
     prometheus:
       enabled: 'METRICS_PROMETHEUS_ENABLED'
+    influx:
+      enabled: 'METRICS_INFLUX_ENABLED'
+      uri: 'METRICS_INFLUX_URI'
```

I think we've mostly converted over our usage of URI to URL.

```suggestion
      url: 'METRICS_INFLUX_URL'
```

---

done